src/app/pages/storage/volumes/datasets/dataset-form/dataset-form.component.ts

```diff
@@ -398,9 +398,9 @@ export class DatasetFormComponent implements Formconfiguration{
       placeholder: helptext.dataset_form_recordsize_placeholder,
       tooltip: helptext.dataset_form_recordsize_tooltip,
       options: [
-        { label: '512', value: '512', disable:true },
-        { label: '1K', value: '1K', disable:true },
-        { label: '2K', value: '2K', disable:true },
+        { label: '512', value: '512', disable:true, hiddenFromDisplay: true},
```

Comment:
These three lines now don't have a trailing space before the last }, so don't look like the lines following.
All of them have leading spaces, so should also have trailing spaces.

Reply:
addressed.